lib/components/Charts/utils/elements.tsx

```diff
@@ -9,6 +9,7 @@ export const xAxisProps = (
   tickLine: false,
   axisLine: false,
   tickMargin: 8,
+  tickCount: config?.tickCount,
```

Comment:
What's the use case for this? Do we really want to make it configurable?

Reply:
We had this use case on the employees' page, we needed to add a chart in a small place that with many y points, caused the chart to look off.

Reply:
Okay! Should we default it to false, perhaps?